AK/HashTable.h

```diff
@@ -258,6 +258,17 @@ class HashTable {
     {
         *this = HashTable();
     }
+    void clear_with_capacity()
+    {
+        __builtin_memset(m_buckets, 0, capacity());
```

Comment:
big oops on my side, this should use `size_in_bytes()`
a Bucket is way larger than one byte...